src/il_ros_hsr/nets/options.py

```diff
@@ -19,8 +19,8 @@
 # ------------------------------------------------------------------------------
 
 # For `prepare_data.py`
-RAW_DATA_SSL = 'ssldata2'
-NEW_DATA_SSL = 'ssldata2_pytorch'
+RAW_DATA_SSL = 'ssldata'
+NEW_DATA_SSL = 'ssldata_pytorch'
 
```

Comment:
@ryanhoque I assume this refers to only the newer data, we'll just ignore the first one and treat your latest one as ssldata?

Reply:
Yup that was the intention